org.upnproutercontrol.UPnPRouterControl.json

```diff
@@ -0,0 +1,75 @@
+{
+   "app-id":"org.upnproutercontrol.UPnPRouterControl",
+   "branch":"unstable",
+   "runtime":"org.gnome.Platform",
+   "runtime-version":"3.38",
+   "sdk":"org.gnome.Sdk",
+   "command":"upnp-router-control",
+   "rename-desktop-file":"upnp-router-control.desktop",
+   "finish-args":[
+      "--share=network",
+      "--socket=session-bus",
+      "--socket=x11",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--filesystem=xdg-config/gtk-3.0"
+   ],
+   "modules":[
+      {
+         "name":"intltool",
+         "sources":[
+            {
+               "type":"archive",
+               "url":"https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz",
+               "md5":"12e517cac2b57a0121cda351570f1e63"
+            }
+         ],
+         "cleanup":[
+            "*"
+         ]
+      },
+      {
+         "name":"upnp-router-control",
+         "buildsystem":"autotools",
+         "config-opts":[
+            "--disable-libcurl"
+         ],
+         "sources":[
+            {
+               "type":"git",
+               "url":"https://git.launchpad.net/upnp-router-control"
```

Comment:
this needs to point to a specific tag or commit per https://github.com/flathub/flathub/wiki/App-Requirements#stable-releases-reproducible-builds